### PR TITLE
Monthly Release Update 2022.08.0

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Updated dev docs, API RNs, and OIE API RNs for the monthly release.
- **Is this PR related to a Monolith release?** Yes, 2022.08.0

### Resolves:

* [OKTA-518646](https://oktainc.atlassian.net/browse/OKTA-518646)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 1]
```diff
@@ -52,6 +64,17 @@ Therefore, when dealing with Custom Roles, these three entities always exist:
 * A principal: Either a Group or a User - known as a Member of the Binding
 * A Resource Set: Identified by its `id`
 
+### Custom Role Assignment for IAM-based Standard Roles
+<ApiLifecycle access="ea" />
+When using IAM-based Standard Roles the same concepts as Custom Roles apply with the following distinctions:
+1. IAM-based standard roles can only be used with predefined Resource Sets.
+2. For both IAM-based Standard Roles and Resource Sets, there is a predefined constant `id`. This `id` will always be the `type` of the Role or Resource Set.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:71]:
```suggestion
2. For both IAM-based Standard Roles and Resource Sets, there is a predefined constant `id`. This `id` is always the `type` of the Role or Resource Set.
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 2]
```diff
@@ -128,11 +152,14 @@ The following are the supported resources.
...
+|                         | All Access Requests               <br><ApiLifecycle access="ea" />  | `orn:$partition$:governance:$orgId$:requests`                                         |                                                                                                                                                         |
 > **Note:** If you use a Role with permissions that don't apply to the resources in the Resource Set, it doesn't affect the admin Role. For example, the `okta.users.userprofile.manage` permission gives the admin no privileges if it is granted to a Resource Set that only includes `https://${yourOktaDomain}/api/v1/groups/${targetGroupId}` resources. If you want the admin to be able to manage the Users within the group, the Resource Set must include the corresponding `https://${yourOktaDomain}/api/v1/groups/${targetGroupId}/users` resource.
 
+> **Note:** Governance resources are currently only supported as part of the [Standard Resource Sets](#standard-resource-sets). These cannot be used to create or update other resource sets.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:161]:
```suggestion
> **Note:** Governance resources are currently only supported as part of the [Standard Resource Sets](#standard-resource-sets). You can't use these to create or update other resource sets.
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 3]
```diff
@@ -25,6 +26,17 @@ The following Role types are provided and supported:
...
 
+### IAM-based Standard Role Types
+<ApiLifecycle access="ea" />
+We also support the following IAM-based standard roles:
+| Role type                                                  | Label                               | Permissions                                   |
+| :--------------------------------------------------------- | :---------------------------------- | :-------------------------------------------- |
+| `ACCESS_CERTIFICATIONS_ADMIN` <ApiLifecycle access="ea" /> | Access Certifications Administrator | `okta.governance.accessCertifications.manage` |
+| `ACCESS_REQUESTS_ADMIN`       <ApiLifecycle access="ea" /> | Access Requests Administrator       | `okta.governance.accessRequests.manage`       |
+IAM-based standard roles could be assigned using [standard](#standard-role-assignment-steps) or [custom role](#custom-role-assignment) assignment operations.
+Similar to other standard roles, these roles are also immutable and cannot be updated or deleted.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:37]:
```suggestion
Similar to other standard roles, these roles are also immutable and can't be updated or deleted.
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 4]
```diff
@@ -149,6 +176,16 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
   https://${yourOktaDomain}/api/v1/groups/${memberGroupId}
   ```
 
+### Standard Resource Sets
+<ApiLifecycle access="ea" />
+The following resource sets are currently supported out of the box and can be used to assign admins only when used with
+their associated roles. Standard resource sets and roles are always identified using their type as id.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:182]:
```suggestion
their associated roles. Standard resource sets and roles are always identified using their type as `id`.
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 5]
```diff
@@ -149,6 +176,16 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
...
+| Resource Set id/type                                              | Applicable Role id/type    | Resources                                              |
+| :---------------------------------------------------------------- | :------------------------- | ------------------------------------------------------ |
+| `ACCESS_CERTIFICATIONS_IAM_POLICY` <ApiLifecycle access="ea" /> | `ACCESS_CERTIFICATIONS_ADMIN` | All Users, All Groups, All Apps, All Access Certifications    |
+| `ACCESS_REQUESTS_IAM_POLICY`       <ApiLifecycle access="ea" /> | `ACCESS_REQUESTS_ADMIN`       | All Users, All Groups, All Access Requests, Access Request App |
+Standard resource sets are managed by Okta only and cannot be updated or deleted.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:187]:
```suggestion
Standard resource sets are managed by Okta only and can't be updated or deleted.
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 6]
```diff
@@ -195,4 +232,8 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
...
-| `okta.workflows.invoke`       <br><ApiLifecycle access="ea" />  | Allows the admin to view and run delegated flows                                                                                                    | All Delegated Flows, a specific Delegated Flow |
+| `okta.governance.accessCertifications.manage` <br><ApiLifecycle access="ea" />  | Allows the admin to view and manage access certification campaigns                                                                  | All Certifications |
+| `okta.governance.accessRequests.manage`  <br><ApiLifecycle access="ea" />  | Allows the admin to view and manage access requests                                                                                       | All Access Requests |
+| `okta.apps.manageFirstPartyApps`  <br><ApiLifecycle access="ea" />  | Allows the admin to manage first-party apps                                                                                       | All Access Requests |
+> **Note:** Governance permissions are currently only supported as part of the [Standard IAM-based Roles](#iam-based-standard-role-types). These cannot be used to create or update other roles.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:238]:
```suggestion
> **Note:** Governance permissions are currently only supported as part of the [Standard IAM-based Roles](#iam-based-standard-role-types). You can't use these to create or update other roles.
```

File: packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md [hunk 7]
```diff
@@ -195,4 +232,8 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
...
+> **Note:** `okta.apps.manageFirstPartyApps` permission is currently only supported as part of some [Standard IAM-based Roles](/docs/concepts/role-assignment/#iam-based-standard-role-types). It cannot be used to create or update other roles.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md:239]:
```suggestion
> **Note:** `okta.apps.manageFirstPartyApps` permission is currently only supported as part of some [Standard IAM-based Roles](/docs/concepts/role-assignment/#iam-based-standard-role-types). You can't use it to create or update other roles.
```

File: packages/@okta/vuepress-site/docs/reference/api/roles/index.md [hunk 8]
```diff
@@ -3064,6 +3193,11 @@ User permissions are only effective with respect to the Group(s) to which the ad
...
+| `okta.governance.accessCertifications.manage` <br><ApiLifecycle access="ea" />  | Allows the admin to view and manage access certification campaigns                                                                  | All Access Certifications |
+| `okta.governance.accessRequests.manage`  <br><ApiLifecycle access="ea" />  | Allows the admin to view and manage access requests                                                                                       | All Access Requests |
+| `okta.apps.manageFirstPartyApps`  <br><ApiLifecycle access="ea" />  | Allows the admin to manage first-party apps                                                                                       | All Access Requests |
+> **Note:** Governance permissions are currently only supported as part of the [Standard IAM-based Roles](/docs/concepts/role-assignment/#iam-based-standard-role-types). These cannot be used to create or update other roles.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/roles/index.md:3199]:
```suggestion
> **Note:** Governance permissions are currently only supported as part of the [Standard IAM-based Roles](/docs/concepts/role-assignment/#iam-based-standard-role-types). You can't use these to create or update other roles.
```

File: packages/@okta/vuepress-site/docs/reference/api/roles/index.md [hunk 9]
```diff
@@ -3064,6 +3193,11 @@ User permissions are only effective with respect to the Group(s) to which the ad
...
+> **Note:** `okta.apps.manageFirstPartyApps` permission is currently only supported as part of some [Standard IAM-based Roles](/docs/concepts/role-assignment/#iam-based-standard-role-types). It cannot be used to create or update other roles.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/roles/index.md:3200]:
```suggestion
> **Note:** `okta.apps.manageFirstPartyApps` permission is currently only supported as part of some [Standard IAM-based Roles](/docs/concepts/role-assignment/#iam-based-standard-role-types). You can't use it to create or update other roles.
```

File: packages/@okta/vuepress-site/docs/reference/api/roles/index.md [hunk 10]
```diff
@@ -3122,9 +3256,12 @@ The following are the supported resources.
...
+| Governance              | All Access Certifications         <br><ApiLifecycle access="ea" />  | `orn:$partition$:governance:$orgId$:certifications`                                   |                                                                                                                                                         |
+|                         | All Access Requests               <br><ApiLifecycle access="ea" />  | `orn:$partition$:governance:$orgId$:requests`                                         |                                                                                                                                                         |
 
 The ID of a resource is unique to the Resource Set, whereas the link that points to the resource is unique for the org. A Group, if used in two Resource Sets, has distinct IDs in each Resource Set but has the same self link in both.
 
+> **Note:** Governance resources are currently only supported as part of the [Standard Resource Sets](/docs/concepts/role-assignment/#standard-resource-sets). These cannot be used to create or update other resource sets.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/roles/index.md:3264]:
```suggestion
> **Note:** Governance resources are currently only supported as part of the [Standard Resource Sets](/docs/concepts/role-assignment/#standard-resource-sets). You can't use these to create or update other resource sets.
```

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 11]
```diff
@@ -4,6 +4,68 @@ title: Okta Identity Engine API Products release notes 2022
...
+
+#### Telephony Inline Hook is LGA in Production
+
+While Okta provides out-of-the-box telephony functionality, many customers need the ability to integrate their existing telecommunications provider with Okta to deliver SMS and Voice messages. The Telephony Inline Hook allows customers to generate One-Time Passcodes within Okta, and then use their existing telecommunications provider to deliver the messages for MFA enrollment/verification, password reset, and account unlock. This allows customers to use their existing telephony solution within Okta, due to the time they've already invested in their existing telephony solution, the need to use a specific regional provider, or simply the desire to maintain flexibility. See [Telephony Inline Hook with Twilio](/docs/guides/telephony-inline-hook/nodejs/main/). <!-- OKTA-518233 -->
+
+#### System Log updates for telephony operations
+
+The `system.operation.rate_limit.violation` event is no longer fired when SMS or Voice messages are blocked due to telephony operational rate limit violations. <!-- OKTA-517838 -->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:43]:
```suggestion
The `system.operation.rate_limit.violation` event is no longer triggered when SMS or Voice messages are blocked due to telephony operational rate limit violations. Instead, telephony `system.sms.send.*` and `system.voice.send.*` events are issued as a `DENY` System Log message. <!-- OKTA-517838 -->
```

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 12]
```diff
@@ -2,6 +2,49 @@
...
+
+Admins can now configure a percentage rate limit capacity for individual API tokens. Previously, when a token rate limit violation occurred, it wasn’t clear which token consumed the limit. Setting a maximum capacity for each token solves this problem and gives admins a new tool to investigate rate limit violations and plan for future deployments. See [API token management](https://help.okta.com/okta_help.htm?id=csh-api). <!-- OKTA-517890 -->
+
+#### Rate Limits dashboard includes API Token data
+
+The Rate Limits dashboard now includes API Token data on the Rate limit usage over time graph. You can select to view bar graph data from API tokens or by IP address to review any spike in traffic. See [bar graph](/docs/reference/rl-dashboard/#bar-graph) and [API rate limits by token](/docs/reference/rate-limits/#api-rate-limits-by-token). <!-- OKTA-498252 -->
+
+#### System Log updates for telephony operations
+
+The `system.operation.rate_limit.violation` event is no longer fired when SMS or Voice messages are blocked due to telephony operational rate limit violations. <!-- OKTA-517838 -->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:33]:
```suggestion
The `system.operation.rate_limit.violation` event is no longer triggered when SMS or Voice messages are blocked due to telephony operational rate limit violations. Instead, telephony `system.sms.send.*` and `system.voice.send.*` events are issued as a `DENY` System Log message. <!-- OKTA-517838 -->
```

**Review by susanharper-okta — APPROVED** [review]
just a couple style nits and the updated system log blurb for telephony. approving to not hold you up

**Comment by brianduffield-okta** [comment]
2022.08.0 released to preview (oktapreview.com:2) as of 7:24 PM Eastern.

**Pull request merged**